backend/plugin/advisor/pg/advisor_naming_fully_qualified.go

```diff
@@ -0,0 +1,300 @@
+}
+
+type FullyQualifiedObjectNameAdvisor struct{}
+
+type FullyQualifiedObjectNameChecker struct {
+	_            *sql.DB
+	adviceList   []advisor.Advice
+	status       advisor.Status
+	title        string
+	line         int
+	isSelectStmt bool
+}
+
+// Visit implements ast.Visitor.
+func (checker *FullyQualifiedObjectNameChecker) Visit(in ast.Node) ast.Visitor {
+	switch node := in.(type) {
+	// Create statement.
+	case *ast.CreateTableStmt:
+		if node.Name != nil {
+			fullyQualifiedName := getFullyQualiufiedObjectName(node.Name)
+			checker.appendAdviceByObjName(fullyQualifiedName)
+		}
+	case *ast.CreateSequenceStmt:
+		if node.SequenceDef.SequenceName != nil {
+			fullyQualifiedName := getFullyQualiufiedObjectName(node.SequenceDef.SequenceName)
+			checker.appendAdviceByObjName(fullyQualifiedName)
+		}
+	case *ast.CreateTriggerStmt:
+		if node.Trigger != nil && node.Trigger.Table != nil {
+			fullyQualifiedName := getFullyQualiufiedObjectName(node.Trigger.Table)
+			checker.appendAdviceByObjName(fullyQualifiedName)
+		}
+
+	// Drop statement.
+	case *ast.DropSequenceStmt:
+		if node.SequenceNameList != nil {
+			for _, seqName := range node.SequenceNameList {
+				fullyQualifiedName := getFullyQualiufiedObjectName(seqName)
+				checker.appendAdviceByObjName(fullyQualifiedName)
+			}
+		}
+	case *ast.DropTableStmt:
+		if node.TableList != nil {
+			for _, table := range node.TableList {
+				fullyQualifiedName := getFullyQualiufiedObjectName(table)
+				checker.appendAdviceByObjName(fullyQualifiedName)
+			}
+		}
+	case *ast.DropIndexStmt:
+		if node.IndexList != nil {
+			for _, index := range node.IndexList {
+				fullyQualifiedName := getFullyQualiufiedObjectName(index)
+				checker.appendAdviceByObjName(fullyQualifiedName)
+			}
+		}
+	case *ast.DropTriggerStmt:
+		if node.Trigger != nil && node.Trigger.Table != nil {
+			fullyQualifiedName := getFullyQualiufiedObjectName(node.Trigger.Table)
+			checker.appendAdviceByObjName(fullyQualifiedName)
+		}
+
+	// Alter statement.
+	case *ast.AlterTableStmt:
+		if node.Table != nil {
+			fullyQualifiedName := getFullyQualiufiedObjectName(node.Table)
+			checker.appendAdviceByObjName(fullyQualifiedName)
+		}
+	case *ast.AlterSequenceStmt:
+		if node.Name != nil {
+			fullyQualifiedName := getFullyQualiufiedObjectName(node.Name)
+			checker.appendAdviceByObjName(fullyQualifiedName)
+		}
+
+	// Insert statement.
+	case *ast.InsertStmt:
+		if node.Table != nil {
+			fullyQualifiedName := getFullyQualiufiedObjectName(node.Table)
+			checker.appendAdviceByObjName(fullyQualifiedName)
+		}
+
+	// Select statement.
+	case *ast.SelectStmt:
+		checker.isSelectStmt = true
+		if node.FieldList != nil {
+			for _, field := range node.FieldList {
+				if fieldDef, ok := field.(*ast.ColumnNameDef); ok {
+					if fieldDef.ColumnName == "*" {
+						break
+					}
+					fullyQualifiedName := getFullyQualiufiedObjectName(fieldDef)
+					checker.appendAdviceByObjName(fullyQualifiedName)
+				}
+			}
+		}
```

Comment:
No need to check the column name here. We should check the `owner of the object`. In the PostgreSQL context, the `owner` means the `schema`. So we should check the object with the schema name or not.